src/redux/bookCategories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const bookCata = 'Categories';
```

Comment:
Hi @LoviMadaan,

Great job on your code submission! I noticed that you used a string instead of an array of categories (initial state: empty) in your slice of state for categories.

To meet the project requirements and enable dynamic rendering of categories based on user input, it's important to use an array. Strings can only hold one value and are not suitable for this purpose.

Can you please update your code to include an array of categories? Let me know if you need any help.

Thanks!

<img width="260" alt="image" src="https://user-images.githubusercontent.com/52934128/233541358-4db3a5ef-6204-476b-b2f5-70d54b108406.png">